GitUI/UserControls/RevisionGrid/Graph/RevisionGraphRevision.cs

```diff
@@ -42,6 +42,8 @@ public void ApplyFlags(RevisionNodeFlags types)
         /// </summary>
         public int Score { get; private set; }
 
+        public void OffsetScore(int offset) => Score += offset;
```

Comment:
`Score` has a decent docs, it would be great to have similar docs for this method too.

Reply:
added